cinema/urls.py

```diff
@@ -1,10 +1,41 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaViewSet,
+    MovieViewSet
+)
+
+
+cinema_hall_list = CinemaViewSet.as_view(actions={
+    "get": "list",
+    "post": "create"
+})
+cinema_hall_detail = CinemaViewSet.as_view(actions={
+    "get": "retrieve",
+    "put": "update",
+    "patch": "partial_update",
+    "delete": "destroy"
+})
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet, basename="movies")
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("genre/", GenreList.as_view(), name="movie-list"),
```

Comment:
The name 'movie-list' for the GenreList path is misleading. Consider renaming it to 'genre-list' to accurately reflect the resource being accessed.